en/index.md

```diff
@@ -14,62 +14,60 @@ This guide contains everything you need to assemble, configure, and safely fly a
 
 :::warning
 This guide is for the _development_ version of PX4 (`main` branch).
-Use the **Version** selector for the current _stable_ version.
+Use the **Version** selector to find the current _stable_ version.
 
 Documented changes since the stable release are captured in the evolving [release note](releases/main.md).
 :::
 
 ## How Do I Get Started?
 
-[Getting Started](getting_started/index.md) should be read by all users!
+[Basic Concepts](../getting_started/px4_basic_concepts.md) should be read by all users!
 It provides an overview of PX4, including features provided by the flight stack (flight modes and safety features) and the supported hardware (flight controller, vehicles, airframes, telemetry systems, RC control systems).
 
-Depending on what you want to achieve, the following tips will help you navigate through this guide:
+If you're just getting started with flying drones then we recommend [multicopters](frames_multicopter/index.md), because they are easier to fly than planes and less complicated to build than VTOLs.
+Once you're familiar with multicopters, then moving onto planes and VTOL is much easier.
 
-**I already have a drone and I just want to fly:**
+Depending on what you want to achieve, the following tips will help you navigate through this guide:
 
-If you have a Ready To Fly (RTF) vehicle that supports PX4:
+### How do get a vehicle that works with PX4?
 
-- [Basic Configuration](config/index.md) explains how to update your firmware to the latest version, calibrate the main sensors (compass, gyro/IMU, airspeed etc.), and setup your remote control and safety features.
-- [Flying](flying/index.md) teaches flight essentials, including where and how to fly safely, and how to debug arming and flight issues. It also provides detailed information about flight modes.
+If you don't have already have Multicopter vehicle then [Complete Vehicles](complete_vehicles_mc/index.md) lists "Ready to Fly" (RTF) pre-build vehicles that you can buy, [Kits](frames_multicopter/kits.md) lists drones that you have to build yourself from a package, and [DIY Builds](frames_multicopter/diy_builds.md) shows some examples of drones that have been build using parts that have been sourced individually.

```

Comment:
```suggestion
If you don't already have a Multicopter vehicle then [Complete Vehicles](complete_vehicles_mc/index.md) lists "Ready to Fly" (RTF) pre-built vehicles that you can buy, [Kits](frames_multicopter/kits.md) lists drones that you have to build yourself from a package, and [DIY Builds](frames_multicopter/diy_builds.md) shows some examples of drones that have been built using parts that were sourced individually.
```
You write better than me so might be garbage.

Reply:
Nope, sadly in this case your writing is much better than mine :-(. Rushed to get it out without proper self-review.